Tank dispencer

maleyvich opened the pull request:
<!--
Читать: https://github.com/TauCetiStation/TauCetiClassic/blob/master/.github/wiki/STYLING_OF_PR.md
-->
## Описание изменений

## Почему и что этот ПР улучшит

## Авторство

<!-- 
В случае порта с другого билда - укажите источник (репозиторий или номер PR-а). 
Если это оригинальный PR - укажите первоисточник/авторство спрайтов и звуков. 
Укажите лицензию для звуков.
-->

## Чеинжлог

<!-- 
В чеинжлог стоит писать изменения, которые будут заметны игрокам. И так, чтобы они были понятны игрокам.
Ключевые слова для чеинжлога: bugfix, rscadd, rscdel, image, sound, spellcheck, tweak, balance, map, performance, experiment

:cl:
 - bugfix: Пофикшен такой-то баг.
 - map: Перемаплен такой-то отсек.
 - image: Обновлен такой-то спрайт.
-->

maleyvich committed:
amogus

maleyvich commented:
![image](https://github.com/user-attachments/assets/286536a7-be50-4660-be4e-99347a62a8d0)
![image](https://github.com/user-attachments/assets/0aa2abd3-b21d-436a-8b9d-1e53f60f2940)

maleyvich committed:
Merge branch 'master' into tank-dispencer

maleyvich commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -37,9 +38,9 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
-	dat += "Oxygen tanks: [oxygentanks] - [oxygentanks ? "<A href='byond://?src=\ref[src];oxygen=1'>Dispense</A>" : "empty"]<br>"
-	dat += "Phoron tanks: [phorontanks] - [phorontanks ? "<A href='byond://?src=\ref[src];phoron=1'>Dispense</A>" : "empty"]"
-	var/datum/browser/popup = new(user, "window=dispenser", "[src]")
+	dat += "Кислород: [oxygentanks] [pluralize_russian(oxygentanks, "баллон", "баллона", "баллонов")] - [oxygentanks ? "<A href='?src=\ref[src];oxygen=1'>Получить</A>" : "empty"]<br>"
```
```suggestion
	dat += "Кислород: [oxygentanks] [pluralize_russian(oxygentanks, "баллон", "баллона", "баллонов")] - [oxygentanks ? "<A href='byond://?src=\ref[src];oxygen=1'>Получить</A>" : "empty"]<br>"
```

maleyvich commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -37,9 +38,9 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
-	dat += "Phoron tanks: [phorontanks] - [phorontanks ? "<A href='byond://?src=\ref[src];phoron=1'>Dispense</A>" : "empty"]"
-	var/datum/browser/popup = new(user, "window=dispenser", "[src]")
+	dat += "Кислород: [oxygentanks] [pluralize_russian(oxygentanks, "баллон", "баллона", "баллонов")] - [oxygentanks ? "<A href='?src=\ref[src];oxygen=1'>Получить</A>" : "empty"]<br>"
+	dat += "Форон: [phorontanks] [pluralize_russian(phorontanks, "баллон", "баллона", "баллонов")] - [phorontanks ? "<A href='?src=\ref[src];phoron=1'>Получить</A>" : "empty"]"
```
```suggestion
	dat += "Форон: [phorontanks] [pluralize_russian(phorontanks, "баллон", "баллона", "баллонов")] - [phorontanks ? "<A href='byond://?src=\ref[src];phoron=1'>Получить</A>" : "empty"]"
```

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -76,7 +78,8 @@
 	name = "phoron tank"
-	desc = "Contains dangerous phoron. Do not inhale. Warning: extremely flammable."
+	cases = list("баллон с фороном", "баллона с фороном", "баллону с фороном","баллон с фороном", "баллоном с фороном", "баллоне с фороном")
+	desc = "Содержит огнеопасный форон. Не вдыхать!"
```
```suggestion
	desc = "Содержит огнеопасный форон. Не вдыхать! Огнеопасно."
```

maleyvich committed:
Apply suggestion from @maleyvich

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -76,7 +78,8 @@
 	name = "phoron tank"
-	desc = "Contains dangerous phoron. Do not inhale. Warning: extremely flammable."
+	cases = list("баллон с фороном", "баллона с фороном", "баллону с фороном","баллон с фороном", "баллоном с фороном", "баллоне с фороном")
+	desc = "Содержит огнеопасный форон. Не вдыхать! Огнеопасно."
```
```suggestion
	desc = "Содержит форон. Не вдыхать! Огнеопасно!"
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -35,7 +36,8 @@
 /obj/item/weapon/tank/anesthetic
 	name = "anesthetic tank"
-	desc = "A tank with an N2O/O2 gas mix."
+	cases = list("баллон для анестезии", "баллона для анестезии", "баллону для анестезии","баллон для анестезии", "баллоном для анестезии", "баллоне для анестезии")
```
в контексте унификации, лучше
```suggestion
	cases = list("баллон с анестетиком", "баллона с анестетиком", "баллону с анестетиком","баллон с анестетиком", "баллоном с анестетиком", "баллоне с анестетиком")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -12,7 +12,8 @@
 	name = "oxygen tank"
-	desc = "A tank of oxygen."
+	cases = list("баллон с кислородом", "баллона с кислородом", "баллону с кислородом","баллон с кислородом", "баллоном с кислородом", "баллоне с кислородом")
+	desc = "Просто баллон с кислородом."
```
По мне лучше так:
```suggestion
	desc = "Баллон с кислородом, ничего необычного."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -35,7 +36,8 @@
 	name = "anesthetic tank"
-	desc = "A tank with an N2O/O2 gas mix."
+	cases = list("баллон для анестезии", "баллона для анестезии", "баллону для анестезии","баллон для анестезии", "баллоном для анестезии", "баллоне для анестезии")
+	desc = "Баллон с закисью азота."
```
Потерял кислород

```suggestion
	desc = "Баллон со смесью закиси азота и кислорода."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -49,7 +51,7 @@
 /obj/item/weapon/tank/anesthetic/small
 	name = "small anesthetic tank"
-	desc = "A small tank with an N2O/O2 gas mix."
+	desc = "Небольшой баллон с закисью азота."
```
```suggestion
	desc = "Небольшой баллон  со смесью закиси азота и кислорода."
```

maleyvich replied:
ответил в дискорде нет такого понятия

maleyvich replied:
```suggestion
	desc = "Небольшой баллон с кислородно-закисноазотной смесью."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -49,7 +51,7 @@
 
 /obj/item/weapon/tank/anesthetic/small
 	name = "small anesthetic tank"
```
Падежи потерял

maleyvich replied:
```suggestion
	name = "small anesthetic tank"
	cases = list("маленький баллон с анестетиком", " маленького баллона с анестетиком", "маленькому баллону с анестетиком"," маленький баллон с анестетиком", " маленьким баллоном с анестетиком", "маленьком баллоне с анестетиком")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -76,7 +78,8 @@
 /obj/item/weapon/tank/phoron
 	name = "phoron tank"
-	desc = "Contains dangerous phoron. Do not inhale. Warning: extremely flammable."
+	cases = list("баллон с фороном", "баллона с фороном", "баллону с фороном","баллон с фороном", "баллоном с фороном", "баллоне с фороном")
```
```suggestion
	cases = list("баллон с фороном", "баллона с фороном", "баллону с фороном", "баллон с фороном", "баллоном с фороном", "баллоне с фороном")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -106,7 +109,8 @@
 /obj/item/weapon/tank/emergency_oxygen
 	name = "emergency oxygen tank"
-	desc = "Used for emergencies. Contains very little oxygen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с кислородом", "аварийного баллона с кислородом", "аварийному баллону с кислородом","аварийный баллон с кислородом", "аварийным баллоном с кислородом", "аварийном баллоне с кислородом")
```
```suggestion
	cases = list("аварийный баллон с кислородом", "аварийного баллона с кислородом", "аварийному баллону с кислородом", "аварийный баллон с кислородом", "аварийным баллоном с кислородом", "аварийном баллоне с кислородом")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -138,7 +142,8 @@
 /obj/item/weapon/tank/emergency_nitrogen
 	name = "emergency nitrogen tank"
-	desc = "Used for Vox-related emergencies. Contains very little nitrogen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с азотом", "аварийного баллона с азотом", "аварийному баллону с азотом","аварийный баллон с азотом", "аварийным баллоном с азотом", "аварийном баллоне с азотом")
```
<img width="1328" height="804" alt="Image" src="https://github.com/user-attachments/assets/06da2b72-7962-4578-bd86-d942307c9a83" />

```suggestion
	cases = list("экстренный баллон с азотом", "экстренного баллона с азотом", "экстренному баллону с азотом", "экстренный баллон с азотом", "экстренным баллоном с азотом", "экстренном баллоне с азотом")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -106,7 +109,8 @@
 	name = "emergency oxygen tank"
-	desc = "Used for emergencies. Contains very little oxygen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с кислородом", "аварийного баллона с кислородом", "аварийному баллону с кислородом","аварийный баллон с кислородом", "аварийным баллоном с кислородом", "аварийном баллоне с кислородом")
+	desc = "Используется для экстренных ситуаций. Содержит немного кислорода на всякий случай..."
```
```suggestion
	desc = "Для экстренных ситуаций. Запас кислорода крайне мал, так что берегите его до самого крайнего случая."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -163,7 +168,8 @@
 	name = "nitrogen tank"
-	desc = "A tank of nitrogen."
+	cases = list("баллон с азотом", "баллона с азотом", "баллону с азотом","баллон с азотом", "баллоном с азотом", "баллоне с азотом")
+	desc = "Просто баллон с азотом."
```
```suggestion
	desc = "Баллон с азотом, ничего необычного."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -138,7 +142,8 @@
 	name = "emergency nitrogen tank"
-	desc = "Used for Vox-related emergencies. Contains very little nitrogen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с азотом", "аварийного баллона с азотом", "аварийному баллону с азотом","аварийный баллон с азотом", "аварийным баллоном с азотом", "аварийном баллоне с азотом")
+	desc = "Используется воксами при неприятных инцидентах. Содержит мало азота, будьте осторожнее..."
```
```suggestion
	desc = "Используется в экстренных ситуациях для воксов. Запас азота крайне мал, так что берегите его до самого крайнего случая."
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -163,7 +168,8 @@
 /obj/item/weapon/tank/nitrogen
 	name = "nitrogen tank"
-	desc = "A tank of nitrogen."
+	cases = list("баллон с азотом", "баллона с азотом", "баллону с азотом","баллон с азотом", "баллоном с азотом", "баллоне с азотом")
```
```suggestion
	cases = list("баллон с азотом", "баллона с азотом", "баллону с азотом", "баллон с азотом", "баллоном с азотом", "баллоне с азотом")
```

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -1,7 +1,8 @@
 /obj/structure/dispenser
 	name = "tank storage unit"
-	desc = "A simple yet bulky storage device for gas tanks. Has room for up to ten oxygen tanks, and ten phoron tanks."
+	cases = list("портативный раздатчик баллонов", "портативного раздатчика баллонов", "портативному раздатчику баллонов", "портативного раздатчика баллонов", "портативным раздатчиком баллонов", "портативном раздатчике баллонов")
```
С чего бы он стал портативным

```suggestion
	cases = list("хранилище баллонов", "хранилища баллонов", "хранилищу баллонов", "хранилище баллонов", "хранилищем баллонов", "хранилище баллонов")
```

maleyvich replied:
```suggestion
	cases = list("портативный раздатчик баллонов", "портативного раздатчика баллонов", "портативному раздатчику баллонов", "портативный раздатчик баллонов", "портативным раздатчиком баллонов", "портативном раздатчике баллонов")
```

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -1,7 +1,8 @@
 	name = "tank storage unit"
-	desc = "A simple yet bulky storage device for gas tanks. Has room for up to ten oxygen tanks, and ten phoron tanks."
+	cases = list("портативный раздатчик баллонов", "портативного раздатчика баллонов", "портативному раздатчику баллонов", "портативного раздатчика баллонов", "портативным раздатчиком баллонов", "портативном раздатчике баллонов")
+	desc = "Простое громоздкое устройство для хранения баллонов. Вмещает в себя до 10 баллонов кислорода и форона."
```
```suggestion
	desc = "Простое и массивное хранилище для газовых баллонов. Вмещает до десяти баллонов с кислородом и до десяти баллонов с фороном."
```

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -51,27 +52,27 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
+			to_chat(user, "<span class='notice'>Вы положили [CASE(I, NOMINATIVE_CASE)] в [CASE(src, NOMINATIVE_CASE)].</span>")
 		else
-			to_chat(user, "<span class='notice'>[src] is full.</span>")
+			to_chat(user, "<span class='notice'>[CASE(src, NOMINATIVE_CASE)] полон.</span>")
```
```suggestion
			to_chat(user, "<span class='notice'>[CASE(src, NOMINATIVE_CASE)] полно.</span>")
```

maleyvich replied:
в дискорде ответили, подходит только под хранилище

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -51,27 +52,27 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
+			to_chat(user, "<span class='notice'>Вы положили [CASE(src, NOMINATIVE_CASE)] в [CASE(src, NOMINATIVE_CASE)].</span>")
 		else
-			to_chat(user, "<span class='notice'>[src] is full.</span>")
+			to_chat(user, "<span class='notice'>CASE(src, NOMINATIVE_CASE)] полон.</span>")
```
```suggestion
			to_chat(user, "<span class='notice'>CASE(src, NOMINATIVE_CASE)] полно.</span>")
```

maleyvich replied:
выше ответ

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -51,27 +52,27 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
 	if(iswrenching(I))
 		if(anchored)
-			to_chat(user, "<span class='notice'>You lean down and unwrench [src].</span>")
+			to_chat(user, "<span class='notice'>Вы открутили [CASE(src, NOMINATIVE_CASE)].</span>")
```
```suggestion
			to_chat(user, "<span class='notice'>Вы открутили [CASE(src, ACCUSATIVE_CASE)].</span>")
```

AzzyDreemurr13 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -51,27 +52,27 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
 			anchored = FALSE
 		else
-			to_chat(user, "<span class='notice'>You wrench [src] into place.</span>")
+			to_chat(user, "<span class='notice'>Вы вкрутили [CASE(src, NOMINATIVE_CASE)] на место.</span>")
```
господи откуда ты взял ВКРУТИЛИ

<img width="660" height="192" alt="Image" src="https://github.com/user-attachments/assets/b7bf53d1-9952-41af-b2b9-40e23808d1b6" />

```suggestion
			to_chat(user, "<span class='notice'>Вы прикрутили [CASE(src, ACCUSATIVE_CASE)] на место.</span>")
```

AzzyDreemurr13 reviewed (changes requested):
Тяжело, переделывай

maleyvich committed:
Update code/game/objects/items/weapons/tanks/tank_types.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Update code/game/objects/items/weapons/tanks/tank_types.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Update code/game/objects/items/weapons/tanks/tank_types.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -35,7 +36,8 @@
 	name = "anesthetic tank"
-	desc = "A tank with an N2O/O2 gas mix."
+	cases = list("баллон с анестетиком", "баллона с анестетиком", "баллону с анестетиком","баллон с анестетиком", "баллоном с анестетиком", "баллоне с анестетиком")
+	desc = "Баллон со смесью закиси азота и кислорода."
```
```suggestion
	desc = "Баллон со смесью закиси азота."
```

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestions from code review

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -49,7 +51,9 @@
 /obj/item/weapon/tank/anesthetic/small
 	name = "small anesthetic tank"
-	desc = "A small tank with an N2O/O2 gas mix."
+	cases = list("маленький баллон с анестетиком", " маленького баллона с анестетиком", "маленькому баллону с анестетиком"," маленький баллон с анестетиком", " маленьким баллоном с анестетиком", "маленьком баллоне с анестетиком")
```
```suggestion
```

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich committed:
Apply suggestion from @maleyvich

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -35,7 +36,8 @@
 	name = "anesthetic tank"
-	desc = "A tank with an N2O/O2 gas mix."
+	cases = list("баллон с анестетиком", "баллона с анестетиком", "баллону с анестетиком","баллон с анестетиком", "баллоном с анестетиком", "баллоне с анестетиком")
+	desc = "Баллон со смесью закиси азота."
```
сюда тоже @maleyvich

AzzyDreemurr13 replied:
```suggestion
	desc = "Баллон с кислородно-закисноазотной смесью."
```

maleyvich committed:
Update code/game/objects/structures/tank_dispenser.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

Pivo24 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -1,7 +1,8 @@
 /obj/structure/dispenser
 	name = "tank storage unit"
-	desc = "A simple yet bulky storage device for gas tanks. Has room for up to ten oxygen tanks, and ten phoron tanks."
+	cases = list("портативный раздатчик баллонов", "портативного раздатчика баллонов", "портативному раздатчику баллонов", "портативного раздатчика баллонов", "портативным раздатчиком баллонов", "портативном раздатчике баллонов")
```
4 падеж - винительный "кого/что"

наблюдаю "портативного раздатчика баллонов"

правильно будет так:

наблюдаю "портативный раздатчик баллонов"

maleyvich replied:
исправленоооо

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich committed:
Apply suggestion from @AzzyDreemurr13

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -49,7 +51,8 @@
 /obj/item/weapon/tank/anesthetic/small
 	name = "small anesthetic tank"
-	desc = "A small tank with an N2O/O2 gas mix."
+	cases = list("маленький баллон с анестетиком", " маленького баллона с анестетиком", "маленькому баллону с анестетиком"," маленький баллон с анестетиком", " маленьким баллоном с анестетиком", "маленьком баллоне с анестетиком")
```
```suggestion
	cases = list("маленький баллон с анестетиком", "маленького баллона с анестетиком", "маленькому баллону с анестетиком", "маленький баллон с анестетиком", "маленьким баллоном с анестетиком", "маленьком баллоне с анестетиком")
```

maleyvich committed:
Apply suggestion from @maleyvich

maleyvich commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -1,7 +1,8 @@
 /obj/structure/dispenser
 	name = "tank storage unit"
-	desc = "A simple yet bulky storage device for gas tanks. Has room for up to ten oxygen tanks, and ten phoron tanks."
+	cases = list("портативный раздатчик баллонов", "портативного раздатчика баллонов", "портативному раздатчику баллонов", "портативный раздатчик баллонов", "портативным раздатчиком баллонов", "портативном раздатчике баллонов")
```
```suggestion
	cases = list("раздатчик баллонов", "раздатчика баллонов", "раздатчику баллонов", "раздатчик баллонов", "раздатчиком баллонов", "раздатчике баллонов")
```

maleyvich committed:
Apply suggestions from code review

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>
Co-authored-by: Victor <104298869+maleyvich@users.noreply.github.com>

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -106,7 +110,8 @@
 /obj/item/weapon/tank/emergency_oxygen
 	name = "emergency oxygen tank"
-	desc = "Used for emergencies. Contains very little oxygen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с кислородом", "аварийного баллона с кислородом", "аварийному баллону с кислородом", "аварийный баллон с кислородом", "аварийным баллоном с кислородом", "аварийном баллоне с кислородом")
```
Бля, я как-то проглядел. Аварийный в падежах:


```suggestion
	cases = list("экстренный баллон с кислородом", "экстренного баллона с кислородом", "экстренному баллону с кислородом", "экстренный баллон с кислородом", "экстренным баллоном с кислородом", "экстренном баллоне с кислородом")
```

AzzyDreemurr13 commented on code/game/objects/items/weapons/tanks/tank_types.dm:
```diff
@@ -106,7 +110,8 @@
 	name = "emergency oxygen tank"
-	desc = "Used for emergencies. Contains very little oxygen, so try to conserve it until you actually need it."
+	cases = list("аварийный баллон с кислородом", "аварийного баллона с кислородом", "аварийному баллону с кислородом", "аварийный баллон с кислородом", "аварийным баллоном с кислородом", "аварийном баллоне с кислородом")
+	desc = "Для экстренных ситуаций. Запас кислорода крайне мал, так что берегите его до самого крайнего случая."
```
Более лаконично
```suggestion
	desc = "Используется в экстренных ситуациях. Запас кислорода крайне мал, так что берегите его до самого крайнего случая."
```

maleyvich committed:
Update code/game/objects/items/weapons/tanks/tank_types.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

maleyvich committed:
Update code/game/objects/items/weapons/tanks/tank_types.dm

Co-authored-by: Azzy <74739991+AzzyDreemurr13@users.noreply.github.com>

AzzyDreemurr13 reviewed (dismissed):
Вроде да

Pivo24 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -105,7 +106,7 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
 					O = new /obj/item/weapon/tank/oxygen(loc)
 				O.loc = loc
-				to_chat(usr, "<span class='notice'>You take [O] out of [src].</span>")
+				to_chat(usr, "<span class='notice'>Вы вытащили [CASE(O, NOMINATIVE_CASE)] из [CASE(src, GENITIVE_CASE)].</span>")
```
В целом норм, чисто тут винительные падежи (вытащили кого/что) поставить вместо именительного, и можно выкатывать

maleyvich replied:
```suggestion
				to_chat(usr, "<span class='notice'>Вы вытащили [CASE(O, ACCUSATIVE_CASE)] из [CASE(src, GENITIVE_CASE)].</span>")
```

Pivo24 commented on code/game/objects/structures/tank_dispenser.dm:
```diff
@@ -117,7 +118,7 @@ ADD_TO_GLOBAL_LIST(/obj/structure/dispenser, tank_dispenser_list)
 					P = new /obj/item/weapon/tank/phoron(loc)
 				P.loc = loc
-				to_chat(usr, "<span class='notice'>You take [P] out of [src].</span>")
+				to_chat(usr, "<span class='notice'>Вы вытащили [CASE(P, NOMINATIVE_CASE)] из [CASE(src, GENITIVE_CASE)].</span>")
```
и тут

maleyvich replied:
```suggestion
				to_chat(usr, "<span class='notice'>Вы вытащили [CASE(P, ACCUSATIVE_CASE)] из [CASE(src, GENITIVE_CASE)].</span>")
```

maleyvich committed:
Update code/game/objects/structures/tank_dispenser.dm

maleyvich committed:
Update code/game/objects/structures/tank_dispenser.dm

AzzyDreemurr13 committed:
пусто

simb11 committed:
куалити оф лайф апдейт фор баклинг (#14480)

* 12345

* Update buckling.dm

* Update buckling.dm

* fix

* Update buckling.dm

* Update buckling.dm

* Update buckling.dm

* Update buckling.dm

* Update buckling.dm

* Update buckling.dm

* Update buckling.dm

AzzyDreemurr13 committed:
Дроны лишились дифузов (#14744)

LudwigVonChesterfield committed:
Portable Generators - Climate Change Edition (#14771)

* portable generators reworked

- now with gas!
- and malfunctions.

* formula fixes.

* add dependency on seconds_per_tick to:

- needed_sheets (doubled time per sheet for all generators as now it represents seconds rather than ticks)
- heat increase
- kaboom probability

* added seconds_per_tick dependency to ambient cooling

* some renames for better clarity

* fixes fuel usage rate calculation

daedrowe committed:
Мелкоприёмник, батарейки, два охлада (#14774)

* hello

* uh huh

* proc

* audible

* span

HyakkimaruLD committed:
Added cylinder rotation mechanics for all revolvers, fix for rus357 (#14818)

* Added cylinder rotation mechanics for all revolvers, fix for rus357

* Replaced probabilistic test with deterministic one (edge ​​cases)

Lexakr committed:
Редизайн кабинета психолога (#14896)

* Редизайн кабинета психолога

* Предложения из комментов

* Удалил камеру из туалета медбея (почему нет?)

* Revert "Удалил камеру из туалета медбея (почему нет?)"

This reverts commit b4961a200beb453e5383c1c22b83b1ea143c668a.

* Чето сломалось от реверта

* Плитка вместо паркета на электростуле

AzzyDreemurr13 committed:
Merge branch 'master' into tank-dispencer
